trk/src/Screens/Home/index.js

```diff
@@ -21,7 +24,44 @@ function HomeScreen(props) {
 
   const logo = require('../../../assets/nagimo-logo.png');
 
-  // API Call to database
+  useEffect(() => {
+    const { onLatestFourTapsUpdate } = TapsApi(); // Assuming this is the method for subscribing to changes
+    const { getClimb } = ClimbsApi();
+
+    const unsubscribe = onLatestFourTapsUpdate(async (querySnapshot) => {
+      // Handle the real-time updates here
+      const climbDetailsPromises = querySnapshot.docs.map(async (doc) => {
+        const tapData = doc.data();
+        const climbSnapshot = await getClimb(tapData.climb);
+        // Extract and format the timestamp
+        let timestamp = tapData.timestamp;
+        if (timestamp.toDate) { // Convert Firebase Timestamp to JavaScript Date
+          timestamp = timestamp.toDate().toLocaleTimeString('en-US', {
+            hour: '2-digit',
+            minute: '2-digit',
+            hour12: true,
+            timeZone: 'America/New_York' // NEW YORK TIME
+          });
+        }
+        return climbSnapshot.exists ? {
+          ...tapData,
+          tapTimestamp: timestamp,
+          tapId: doc.id,
+          ...climbSnapshot.data(),
+        } : null;
+      });
+
+
+
+      const newClimbsHistory = (await Promise.all(climbDetailsPromises))
+        .filter(tap => tap !== null);
+
+      setClimbsHistory(newClimbsHistory);
+    });
+
+    // Make sure to unsubscribe from the listener when the component unmounts
+    return () => unsubscribe();
+  }, []);
```

Comment:
New UseEffect responsible for loading the **Latest Gym Taps** onto the screen, and adding a listener to the list.

Reply:
Also, converting the timestamp (in Firebase format) to USA - America - New York timezone